pepper-apis/studybuilder-cli/studies/shared/family-history/snippets/family-history-section-intro.conf

```diff
@@ -52,7 +52,7 @@
           {
             "language": "en",
             "text": """
-              Please do not hesitate to contact our team with questions regarding this survey."""
+              Please do not hesitate to contact our team at <a href="mailto:info@OSproject.org" class='maincolor'>info@OSproject.org</a> or 651-602-2020 with any questions about this survey."""
```

Comment:
Please add more descriptive class name:  e.g. emailColor

Reply:
Please revert this change